src/utils/calculations.ts

```diff
@@ -32,3 +32,22 @@ export function calculateUnderlyingAmount(props: CalculateUnderlyingAmountProps)
   const ONE_UNIT = toBN('10').pow(underlyingTokenDecimals);
   return amount.times(sharePrice).times(ONE_UNIT).toFixed(0);
 }
+
+export function computeSummaryData(labs: Pick<GeneralLabView, 'apyData' | 'DEPOSIT'>[]): SummaryData {
+  const { totalDeposits, totalEarnings } = labs.reduce(
+    ({ totalDeposits, totalEarnings }, { DEPOSIT: { userDepositedUsdc }, apyData }) => ({
+      totalDeposits: totalDeposits.plus(userDepositedUsdc),
+      totalEarnings: totalEarnings.plus(toBN(userDepositedUsdc).times(apyData)),
+    }),
+    {
+      totalDeposits: new BigNumber(0),
+      totalEarnings: new BigNumber(0),
```

Comment:
```suggestion
      totalDeposits: toBN(0),
      totalEarnings: toBN(0),
```
We always try to use our function just so its also easier to replace in the future in case is needed.

Reply:
Can't believe I forgot to use that 🤦 

https://github.com/yearn/yearn-finance-v3/pull/562/commits/60264391382a0ce9cd32a144131835cd86aa87e1